docker-compose.yml

```diff
@@ -33,8 +35,8 @@ services:
        - 9000:9000
      depends_on:
        - "mongodb"
-       - "webapp"
-     command: ["-Dlogger.play=ERROR","-Dlogger.application=ERROR","-Dzeta.webapp.host=webapp", "project server", "run"]
+       - "frontend"
+     command: ["-Dlogger.play=ERROR","-Dlogger.application=ERROR","-Dzeta.webapp.host=frontend", "project server", "run"]
```

Comment:
@pat-St does the backend still need these dependencies + parameter

Reply:
i don´t know. never checked it.